code.user.js

```diff
@@ -109,6 +109,63 @@
         }
     }
 
+    function request(url, options, callback) {
+        let delayBetweenRequests = 300;
+        let requestStorageHash = 'request:last';
+
+        if (url.startsWith('https://steamcommunity.com/market/')) {
+            requestStorageHash = 'request:last:steamcommunity.com/market';
```

Comment:
session storage is probably enough here, and I'd prefix the strings with "see" or something.

Reply:
done